### PR TITLE
Fixtures tab

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?168592#942436

enchantress @esoergel

### CODE REVIEW

File: corehq/apps/fixtures/interface.py [hunk 1]
```diff
@@ -66,7 +71,15 @@ def report_context(self):
...
-        context.update({"selected_table": self.table.get("table_id", "")})
+        context.update({
+            "selected_table": self.table.get("table_id", ""),
+            'active_tab': ProjectDataTab(
+                self.request,
+                self.slug,
+                domain=self.domain,
+                couch_user=self.request.couch_user,
+                project=self.request.project
+        )})
```
Comment (dimagimon) [inline corehq/apps/fixtures/interface.py]: E122 continuation line missing indentation or outdented

File: corehq/apps/fixtures/interface.py [hunk 2]
```diff
@@ -99,8 +112,18 @@ class FixtureEditInterface(FixtureInterface):
...
         context.update(types=self.data_types)
+        context.update({
+            'active_tab': ProjectDataTab(
+                self.request,
+                self.slug,
+                domain=self.domain,
+                couch_user=self.request.couch_user,
+                project=self.request.project
+            )
+            }
```
Comment (dimagimon) [inline corehq/apps/fixtures/interface.py]: E123 closing bracket does not match indentation of opening bracket's line

File: corehq/apps/fixtures/interface.py [hunk 3]
```diff
@@ -17,6 +18,10 @@ class FixtureInterface(FixtureViewMixIn, GenericReportView):
     exportable = False
     needs_filters = False
 
+    @classmethod
+    def display_in_dropdown(cls, domain=None, project=None, user=None):
+        return True
```
Comment (esoergel) [inline corehq/apps/fixtures/interface.py]: I thought the ticket said to not display in the dropdown?

Comment (emord) [inline corehq/apps/fixtures/interface.py]: Yeah. I have bad reading comprehension apparently.

**Pull request merged**